HPXMLtoOpenStudio/resources/electric_panel.rb

```diff
@@ -0,0 +1,251 @@
+# frozen_string_literal: true
+
+# Collection of methods related to electric panel load calculations.
+module ElectricPanel
+  # Calculates load-based capacity and breaker spaces for an electric panel.
+  #
+  # @param hpxml_header [HPXML::Header] HPXML Header object (one per HPXML file)
+  # @param hpxml_bldg [HPXML::Building] HPXML Building object representing an individual dwelling unit
+  # @param electric_panel [HPXML::ElectricPanel] Object that defines a single electric panel
+  # @return [nil]
+  def self.calculate(hpxml_header, hpxml_bldg, electric_panel)
+    capacity_types = []
+    capacity_total_watts = []
+    capacity_total_amps = []
+    capacity_headroom_amps = []
+    hpxml_header.service_feeders_load_calculation_types.each do |service_feeders_load_calculation_type|
+      if service_feeders_load_calculation_type.include?('Load-Based')
```

Comment:
Huh? Why not compare to the HPXML constant?

Reply:
I was trying to anticipate future calculation types being added. I'll make the changes.